assertk/src/jvmMain/kotlin/assertk/assertions/any.kt

```diff
@@ -130,7 +130,7 @@ private fun <T> Assert<T>.isDataClassEqualToImpl(expected: T, kclass: KClass<*>?
 fun <T : Any> Assert<T>.isEqualToIgnoringGivenProperties(other: T, vararg properties: KProperty1<T, Any?>) {
     all {
         for (prop in other::class.members) {
-            if (prop is KProperty1<*, *> && !properties.contains(prop)) {
+            if (prop is KProperty1<*, *> && !properties.contains(prop) && prop.isAccessible) {
```

Comment:
Do you think is it the case to add a proper test for this case, if not already provided? WDYT? Thanks

Reply:
Yes, an extra test on a class with a private var would be good.